tests/pyoozie/test_tags.py

```diff
@@ -833,3 +833,94 @@ def test_workflow_app_inherit_parent_error(request):
     app.assert_node("/action[@name='action-resolve_a']/error", to='action-error')
     app.assert_node("/action[@name='action-error']/error", to='end')
     app.assert_node("/action[@name='action-error']/ok", to='kill-error')
+
+
+def test_workflow_app_empty_decision_entities():
+    with pytest.raises(AssertionError) as assertion_info:
+        tags.Decision(
+            default=None,
+            choices={
+                '${wf:lastErrorNode() eq null}': tags.Action(
```

Comment:
Are users of this library going to need to know how to express conditionals like this? 

It might make sense to abstract this away.

Reply:
Yes, that's a [JSP-EL](https://docs.oracle.com/javaee/1.4/tutorial/doc/JSPIntro7.html) with Oozie entities. There's enough available variation that I'd like to not make an abstraction for this. Instead, on top of this API, I'd like to provide "recipes" for common problems and provide the required JSP-EL in the future.